content/php-client/getting-started.md

```diff
@@ -17,10 +17,17 @@ We use HTTPPlug as the HTTP client abstraction layer. If you want to know more a
 In this example, we will use [Guzzle](https://github.com/guzzle/guzzle) v6 as the HTTP client implementation.
 
 Run the following command to require the libraries in your project:
+
+- Community Edition 
```

Comment:
Can you use subtitle instead of bullet point? It will be much clearer.

Reply:
I didn't want because it is pretty nice with bullet point. Moreover, it does not worth a title for a single command. You can checkout and generate to see the difference

Reply:
I did and it felt weird. But I assume it's personal taste. It's even weirder the second time where you have two code blocks separated by a sentence. The hierarchy of the information is really difficult to get.

Reply:
Ok I did the fix